docs/sql-guide/functions/function-identifier.md

```diff
@@ -0,0 +1,48 @@
+---
+title: IDENTIFIER()
+layout: default
+parent: Functions
+grand_parent: SQL guide
+---
+
+# IDENTIFIER() function
+
+`IDENTIFIER()` function generates unique IDs for the provided index.
```

Comment:
I'd add something more to this because this is actually beneficial and means users don't need to manually add IDs e.g.,

```
The IDENTIFIER() function automatically generates a unique row `_id` for a provided index.
```

Reply:
Another question: will IDENTIFIER() work with the BULK INSERT statement? 
Or is this strictly for INSERT?
Would it work with SELECT statements or when creating views?